src/ui/view/governance/details/details.tsx

```diff
@@ -48,7 +48,7 @@ const Paragraph: FC<ParagraphProps> = ({ paragraph, theme }) => {
   const isTextTooLong = useMemo(
     () =>
       paragraphRef.current && paragraphRef.current.offsetHeight < paragraphRef.current.scrollHeight,
-    [paragraphRef.current]
```

Comment:
Good catch. We can refactor it using useCallback [as suggested here](https://legacy.reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node) to remove this unnecessary ref dependency in another PR.

Reply:
All right. Can you make the change? thanks :)

Reply:
Sure!

Reply:
Done by d05aa339492b70d12afe8ed41e05dc8e03cb8d73.